bluesky/stack/synthetic.py

```diff
@@ -311,35 +314,7 @@ def process(command, numargs, cmdargs, sim, traf, scr, cmd):
             except:
                 scr.echo('Syntax error')
                 scr.echo(commandhelp)      
-                
-

```

Comment:
All the stuff below that was deleted because it doesn't work in the current BlueSky.